test/examples/ucp_hello_world.c

```diff
@@ -329,6 +332,14 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
         goto err;
     }
 
+    test_str = (char*) calloc(test_string_length + 1, sizeof(char));
```

Comment:
The real data length to send will be `test_string_length + 1`. I think it is better to use the same data length as the user requested. If the user wants string length 256, it should be 256 (not 257).
Could you please adjust data size to the user request (remove `+ 1`)?